openlabcmd/requirements.txt

```diff
@@ -1,6 +1,7 @@
 configparser
 enum;python_version=='2.7'
```

Comment:
not related to this patch:  enum can be removed, since openlabcmd is only for py3 now.